benchmarks/luafun/bench.lua

```diff
@@ -46,21 +45,20 @@ function init()
                 break
             end
         end
-
         trace.lastlua = lastlua
         if lastlua then
             trace.stoploc = bor(lshift(lastlua.id, 16), trace.stoppc)
         end
 
+        -- Map trace ids to a consistent increasing value based on there order in the log
         local id = trace.id
+        -- Use the current seen successful trace count as the id for both aborted and successful traces

```

Comment:
I'm not sure how to parse "seen successful".

Reply:
The array the loop is going through is a mix of successful traces and aborted traces. tracecount is the current number of successful traces seen in the loop so far

Reply:
@ltratt what do you think it should say

Reply:
Something like "Use the current successful trace as count as..." maybe?

Reply:
what about "Use the current number of seen successful traces as the id for both aborted and successful traces" ?

Reply:
"Seen successful" just doesn't parse, to my ear at least -- you need to use a different phrase for that part.

Reply:
"Use the total number successful traces seen so far as the id for both aborted and successful traces" ?

Reply:
Works for me!

Reply:
fixed